iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
-// Copyright (c) 2021 - 2022 by Apex.AI Inc. All rights reserved.
+// Copyright (c) 2022 - 2023 by Apex.AI Inc. All rights reserved.
```

Comment:
I think a typo slipped in:

```suggestion
// Copyright (c) 2021 - 2023 by Apex.AI Inc. All rights reserved.
```